applications/launchpad/gui-react/src/containers/MiningContainer/MiningBoxMerged/index.tsx

```diff
@@ -12,12 +12,31 @@ import MiningBox from '../MiningBox'
 import { MiningBoxStatus } from '../MiningBox/types'
 
 import { Container } from '../../../store/containers/types'
+
+import t from '../../../locales'
+
 // import SetupMerged from './SetupMerged'
 import SetupMergedWithForm from './SetupMergedWithForm'
+import {
+  BestChoiceTagIcon,
+  BestChoiceTagText,
+  StyledBestChoiceTag,
+} from './styles'
+
+const BestChoiceTag = () => {
+  return (
+    <StyledBestChoiceTag>
+      <BestChoiceTagText>{t.common.phrases.bestChoice} </BestChoiceTagText>
+      <BestChoiceTagIcon>💪</BestChoiceTagIcon>
+    </StyledBestChoiceTag>
+  )
+}
```

Comment:
The emoji was too large comparing to the text, so I had to wrap this with some styled `span`s